ios/RPCModule.swift

```diff
@@ -186,12 +165,16 @@ class RPCModule: NSObject {
 
   func saveWalletInternal() throws {
     let walletEncodedString = saveToB64()
```

Comment:
The same thing here, actually here happened our problem in IOS (3 users affected).